DESCRIPTION

```diff
@@ -25,6 +25,9 @@ LazyLoad: yes
 Roxygen: list(wrap = FALSE)
 Depends:
     R (>= 2.7)
+Imports:
+    R6,
```

Comment:
Is R6 necessary for the new reader signature?  If there is a way to do the same thing with the standard S4 objects I would prefer that.

Reply:
Like I said `R6` is an indirect dependency of the `tidyverse` through `dplyr`. There is always a way to do it in S4, but it takes more effort, is a lot less concise and probably harder to understand. The definition of this class, including the formatting and printing function takes 15 lines of code when I include the lines with only parentheses. `R6` is really lightweight, and will be installed on many systems already. I really think if we are going to convert to the `tidyverse` internally this is actually more consistent then trying to implement in `base` functions only.